services/121-service/src/migrate-visa/migrate-visa.service.ts

```diff
@@ -321,7 +388,10 @@ export class MigrateVisaService {
       from
         "121-service"."intersolve_visa_customer" i
       left join "121-service".registration r on
-        r.id = i."registrationId" WHERE "programId" = ${programId}`,
+        r.id = i."registrationId" 
+      LEFT JOIN "121-service"."intersolve_migration_progress" imp ON r."referenceId" = imp."referenceId"
```

Comment:
So if we have to run this a second time because of errors. We manually have to clean them from this table?